src/main/java/gnu/io/RXTXPort.java

```diff
@@ -1190,20 +1187,15 @@ public void flush() throws IOException
 	*/
 		public synchronized int read() throws IOException
 		{
-			log.trace( "RXTXPort:SerialInputStream:read() called");
 			if ( fd == 0 ) throw new IOException();
 			if ( monThreadisInterrupted )
 			{
-				log.debug( "+++++++++ read() monThreadisInterrupted" );
+				log.debug( "read(): monThreadisInterrupted" );
```

Comment:
I think this is one of many good candidates for being replaced with a more sane, human-readable message. The presence of the method name in the output can be controlled by the log message formatter, and “monThreadisInterrupted” is fairly inscrutable to anyone not reading along with the code. That said, there are enough other messages which need rewriting that I think trying to do that in this PR (either yours or mine) will quickly spiral into a mess of highly subjective bikeshedding, so let's leave it for now.